scripts/change-fonts-cjk.sh

```diff
@@ -0,0 +1,115 @@
+    for font in $CJKLOCALES; do
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Regular.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Regular.otf"; fi
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Bold.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Bold.otf"; fi
+      sed -i "s/Noto Sans CJK `echo ${font} | tr [:lower:] [:upper:]`/Noto Sans CJK KR/g" ./style/fonts.mss
+    done
+    download "${FONTDIR}/NotoSansCJKkr-Regular.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/Korean/NotoSansCJKkr-Regular.otf"
+    download "${FONTDIR}/NotoSansCJKkr-Bold.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/Korean/NotoSansCJKkr-Bold.otf"
+    break
+  elif [ "$cjklang" = "3" ]; then
+    for font in $CJKLOCALES; do
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Regular.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Regular.otf"; fi
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Bold.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Bold.otf"; fi
+      sed -i "s/Noto Sans CJK `echo ${font} | tr [:lower:] [:upper:]`/Noto Sans CJK SC/g" ./style/fonts.mss
+    done
+    download "${FONTDIR}/NotoSansCJKsc-Regular.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/SimplifiedChinese/NotoSansCJKsc-Regular.otf"
+    download "${FONTDIR}/NotoSansCJKsc-Bold.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/SimplifiedChinese/NotoSansCJKsc-Bold.otf"
+    break
+  elif [ "$cjklang" = "4" ]; then
+    for font in $CJKLOCALES; do
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Regular.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Regular.otf"; fi
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Bold.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Bold.otf"; fi
+      sed -i "s/Noto Sans CJK `echo ${font} | tr [:lower:] [:upper:]`/Noto Sans CJK TC/g" ./style/fonts.mss
+    done
+    download "${FONTDIR}/NotoSansCJKtc-Regular.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/TraditionalChinese/NotoSansCJKtc-Regular.otf"
+    download "${FONTDIR}/NotoSansCJKtc-Bold.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/TraditionalChinese/NotoSansCJKtc-Bold.otf"
+    break
+  elif [ "$cjklang" = "5" ]; then
+    for font in $CJKLOCALES; do
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Regular.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Regular.otf"; fi
+      if [ -e "${FONTDIR}/NotoSansCJK${font}-Bold.otf" ]; then rm "${FONTDIR}/NotoSansCJK${font}-Bold.otf"; fi
+      sed -i "s/Noto Sans CJK `echo ${font} | tr [:lower:] [:upper:]`/Noto Sans CJK HK/g" ./style/fonts.mss
+    done
+    download "${FONTDIR}/NotoSansCJKhk-Regular.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/TraditionalChineseHK/NotoSansCJKhk-Regular.otf"
+    download "${FONTDIR}/NotoSansCJKhk-Bold.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/TraditionalChineseHK/NotoSansCJKhk-Bold.otf"
+    break
+  fi
+done
+
+echo "\nDone."
+echo "Execute 'carto project.mml > mapnik.xml' to apply the change."
```

Comment:
Don't prescribe a particular way of using the MML - this step is not relevant if using Kosmtik or something else that reads CartoCSS directly.